doc/classes/Viewport.xml

```diff
@@ -149,6 +159,14 @@
 			<param index="0" name="event" type="InputEvent" />
 			<param index="1" name="in_local_coords" type="bool" default="false" />
 			<description>
+				Triggers the given [InputEvent] in this [Viewport]. This can be used to pass input events between viewports, or to locally apply inputs that were sent over the network or saved to a file.
+				If [param in_local_coords] is set to [code]true[/code], the given [param event] will be transformed to the local coordinate space of the [Viewport].
```

Comment:
This formulation might be misleading. I suggest: "If [code]in_local_coords[/code] is [code]false[/code], the events position is in the embedders coordinates and will be converted to viewport coordinates. If [code]in_local_coords[/code] is [code]true[/code], the events position is in viewport coordinates."